src/Markdig.Wpf.Shared/Styles.cs

```diff
@@ -64,6 +64,11 @@ public static class Styles
         public static ComponentResourceKey QuoteBlockStyleKey { get; } = new ComponentResourceKey(typeof(Styles), nameof(QuoteBlockStyleKey));
 
         /// <summary>
+        /// Resource Key for the TaskListStyle.
+        /// </summary>
+        public static ComponentResourceKey TaskListStyleKey { get; } = new ComponentResourceKey(typeof(Styles), nameof(TaskListStyleKey));
```

Comment:
TaskList is an extension, we might need to move all extension-related code/styles elsewhere. Not critical for this PR, we can do it later.